src/apify_client/clients/resource_clients/actor_collection.py

```diff
@@ -36,11 +37,12 @@ def list(
             limit: How many Actors to list.
             offset: What Actor to include as first when retrieving the list.
             desc: Whether to sort the Actors in descending order based on their creation date.
+            sort_by: Field to sort the results by. Allowed values are "createdAt" and "lastRunStartedAt".
```

Comment:
This is redundant, and not good for maintainability.

```suggestion
            sort_by: Field to sort the results by.
```